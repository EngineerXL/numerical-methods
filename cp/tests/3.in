3640410640386150363222563
2014108583705233203496292
