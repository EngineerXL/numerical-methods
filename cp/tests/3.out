7332182319013610338856644332591813842423741236396
